tests/e2e/instrumentation-dotnet/00-annotate-namespace.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # Annotate the namespace to allow the application to run using an specific group and user in OpenShift
+  # https://docs.openshift.com/dedicated/authentication/managing-security-context-constraints.html
+  # This annotation has no effect in Kubernetes
+  - command: kubectl annotate namespace ${NAMESPACE} openshift.io/sa.scc.uid-range=1000/1000 --overwrite
```

Comment:
where is the `NAMESPACE` set? Is it set by kuttl and does it represent the test namespace?

Reply:
Instead of adding a new kuttle file and step could we add the annotation directly to manifest that creates the namespace (my-other-ns) - https://github.com/open-telemetry/opentelemetry-operator/blob/1c10a541454bcad27c7b276ec770355ddf793b9a/tests/e2e/instrumentation-java-other-ns/02-install-instrumentation.yaml#L4

Reply:
> where is the `NAMESPACE` set? Is it set by kuttl and does it represent the test namespace?

Yes, it's set by KUTTL.

Reply:
> Instead of adding a new kuttle file and step could we add the annotation directly to manifest that creates the namespace (my-other-ns) -
> 
> https://github.com/open-telemetry/opentelemetry-operator/blob/1c10a541454bcad27c7b276ec770355ddf793b9a/tests/e2e/instrumentation-java-other-ns/02-install-instrumentation.yaml#L4

I'll append the namespace annotation to the step where the collector is deployed. The annotated namespace is the one that is created automatically by KUTTL, not the one created as one step in the test.